docs/basics/ExampleTodoList.md

```diff
@@ -328,10 +319,7 @@ const mapDispatchToProps = (dispatch, ownProps) => {
   }
 }
 
-const FilterLink = connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(Link)
+const FilterLink = connect(mapStateToProps, mapDispatchToProps)(Link)
```

Comment:
Same thing here. In lieu of a decorator, the multiple lines for `connect` calls are easier to read through.